src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, Area, Info {

```

Comment:
Remember that abstract classes should not be used to set behavior for classes. Make sure that the use of abstract class here is justified and that the behavior could not be better represented by an interface.